### PR TITLE
Feature/cleaning

### LINKED ISSUE #30: Refactoring to include hooks

### PR DESCRIPTION
**Pull request by asiisii**
## Description
Please include a summary of the change and which issue is fixed. Please also include relevant screenshots of your code being deployed. 
- Refactor class components to functional components to use hooks 

Fixes # (issue)
- close #30 

## Type of change
Please delete options that are not relevant.
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Refactoring

## Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

File: src/components/PokemonDetails/PokemonDetails.js [hunk 1]
```diff
@@ -20,6 +20,16 @@ const PokemonDetails = ({pokiId, caught, favorite}) => {
   const [pokemonDetails, setPokemonDetails] = useState('')
   // const [id, setId] = useState(pokiId)
   const [error, setError] = useState('')
+  const fetchSinglePokemonInfo = async () => {
+    try {
+      const fetchedPokemonDetails = await fetchPokemonData(`/${pokiId}`)
+      // this.setState({ pokemonDetails: getPokemonDetails(fetchedPokemonDetails)})
+      setPokemonDetails(getPokemonDetails(fetchedPokemonDetails))
```
Comment (Thee-Dust) [inline src/components/PokemonDetails/PokemonDetails.js:17]: i like the use of hooks

**Review by Thee-Dust — APPROVED** [review]
LGTM!

File: src/components/App/App.js [hunk 2]
```diff
@@ -1,85 +1,73 @@
...
+const App = () => {
+  const [pokemons, setPokemons] = useState([])
+  const [caughtPokemon, setCaughtPokemon] = useState([])
+  const [error, setError] = useState('')
+  
+  const catchPokemon = (pokemonName) => {
+    if(!caughtPokemon.includes(pokemonName)) {
+      setCaughtPokemon([...caughtPokemon, pokemonName])
+    } else {
+     setCaughtPokemon(caughtPokemon.filter(poki => {
```
Comment (rhen92) [inline src/components/App/App.js:23]: Great way to implement hooks here to catch favorite pokemon.

**Comment by rhen92** [comment]
Great job cleaning up the code!

**Pull request merged**